src/main/java/core/basesyntax/dao/FruitDaoImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.storage.Storage;
+import java.util.Map;
+
+public class FruitDaoImpl implements FruitDao {
+
+    @Override
+    public Map<String, Integer> getFruits() {
+        return Storage.STORAGE;
+    }
```

Comment:
We have access to Storage. I believe this is redundant.

Reply:
Yeah, just didn't want a straight access to storage, and did all storage manipulations in FruitDao.